helm/application-namespace-prep/tests/authenticator-rolebinding_test.yaml

```diff
@@ -0,0 +1,66 @@
+suite: test authenticator-rolebinding
```

Comment:
Perhaps rename this file to `authenticator_rolebinding_test.yaml`? We should be consistent with using all `-` OR all `_` within a single file name. I *think* that the source files for `helm-unittest` require `_test` at the end, otherwise my preference would be to use hyphens.

Reply:
You are correct, it must be an underscore, just tried it. Will go with the underscores.